docs/api/base-window.md

```diff
@@ -144,11 +144,6 @@ _**Note**: There is a subtle difference between the behaviors of `window.onbefor
 Emitted when the window is closed. After you have received this event you should
 remove the reference to the window and avoid using it any more.
 
-#### Event: 'session-end' _Windows_
```

Comment:
One thing I missed is that we probably don't want to outright remove this event if it can be avoided.

@savely-krasovsky were you able to give @itsananderson's suggestion a try?
https://github.com/electron/electron/pull/44598#discussion_r1842998014

Reply:
@samuelmaddock could you approve my PR if everything is ok now?